### PR TITLE
Add enumpart, map, crsg, adjacency, data, vignette, website, compactness, metrics, and partition distances

### PR DESCRIPTION
**Pull request by christopherkenny**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -9,16 +9,16 @@ Authors@R: c(
...
-    algorithms.  The package allows for the implementation of various constraints in 
+Description: Enables researchers to sample redistricting plans from a pre-specified
+    target distribution using Sequential Monte Carlo and Markov Chain Monte Carlo
+    algorithms.  The package allows for the implementation of various constraints in
     the redistricting process such as geographic compactness and population parity
-    requirements. Tools for analysis such as computation of various summary statistics 
-    and plotting functionality are also included. The package implements methods 
+    requirements. Tools for analysis such as computation of various summary statistics
+    and plotting functionality are also included. The package implements methods
     described in Fifield, Higgins, Imai and Tarr (2020) <doi: 10.1080/10618600.2020.1739532>,
```
Comment (bfifield) [inline DESCRIPTION:18]: also Fifield, Imai, Kawahara, Kenny (2020)

File: NAMESPACE [hunk 2]
```diff
@@ -2,21 +2,34 @@
...
+export(redist.parity)
+export(redist.prep.enumpart)
+export(redist.read.enumpart)
 export(redist.rsg)
+export(redist.run.enumpart)
 export(redist.samplepart)
 export(redist.segcalc)
 export(redist.smc)
 export(redist.smc_is_ci)
+import(ggplot2)
```
Comment (bfifield) [inline NAMESPACE:31]: Is it necessary to import all of `ggplot` and all of `sf`? Can we import just specific functions to conserve package size?

Comment (christopherkenny) [reply NAMESPACE:31]: Yeah, these can be broken down into only the used functions. They will need to be in Suggests, however, as they are used in the current vignette. Moving them will still reduce the size.

**Comment by bfifield** [comment]
There's a `.RData` file in the main directory, which should probably be deleted.

File: NAMESPACE [hunk 3]
```diff
@@ -57,6 +61,24 @@ importFrom(parallel,stopCluster)
 importFrom(readr,read_lines)
 importFrom(readr,write_delim)
 importFrom(servr,make)
+importFrom(sf,"st_crs<-")
```
Comment (bfifield) [inline NAMESPACE:64]: Is this an error that it is importing something called `st_crs<-`? Might be worth double-checking that the @importFrom isn't picking up an extra character.

File: R/compactness.R [hunk 4]
```diff
@@ -97,7 +97,9 @@
 #' 
 #' @importFrom tibble tibble
 #' @importFrom magrittr %>%
-#' @import sf 
+#' @importFrom sf st_cast st_bbox st_centroid st_within st_point_on_surface st_coordinates
+#' @importFrom sf st_linestring st_intersection st_area st_crs st_is_longlat st_length
+#' @importFrom sf st_convex_hull st_crs<- st_geometry st_distance st_union
```
Comment (bfifield) [inline R/compactness.R:102]: Ah yep - here it is. Should the `<-` be removed before `st_crs`?

File: NAMESPACE (hunk 3 above)
Comment (christopherkenny) [reply NAMESPACE:64]: Yes, that is a separate function used in creating the edge overlay for the map functionality.

**Pull request merged**